sys/log/util/pkg.yml

```diff
@@ -0,0 +1,34 @@
+#
```

Comment:
Does `sys/log/util` itself need to be a package?  It doesn't seem to contain any code of its own.

Reply:
done